docs/source/tensor_attributes.rst

```diff
@@ -71,6 +71,17 @@ Via a string and device ordinal:
     >>> torch.device('cpu', 0)
     device(type='cpu', index=0)
 
+.. note::
+   The :class:`torch.device` argument in functions can generally be substituted with a string.
+   This allows for fast prototyping of code.
+
+   >>> # Example of a function that takes in a torch.device
+   >>> cuda1 = torch.device('cuda:1')
+   >>> torch.randn((2,3), device=cuda1)
+
+   >>> # You can substitute the torch.device with a string
+   >>> torch.randn((2,3), 'cuda:1')
```

Comment:
When you put things like this I can't see why any user would prefer to use the `torch.device` way, since it gives you the same thing but more code. Can you please mention that it's not recommended, because perf?